### PR TITLE
Finish setting goals

### PR DESCRIPTION
**Pull request by hshiah**
Finish command for setting goals
Fix bug of finding command

### CODE REVIEW

**Review by ryan1604 — COMMENTED** [review]
LGTM, just a few minor points
Maybe consider adding empty spaces to make code less cluttered and more readable
Can also consider making Arraylist for goal and reminder public so there won't be a need for additional methods

**Review by NeoMinWei — APPROVED** [review]
line 29 of findcommand could be wrapped 
other than that LGTM

**Review by wwweert123 — APPROVED** [review]
LGTM  But maybe in your FindCommand.java file, you can add another abstraction for formating the string to be added to the foundedFinancialList and foundedWatchList

File: src/main/java/seedu/financialplanner/commands/FindCommand.java [hunk 1]
```diff
@@ -25,8 +25,9 @@ public void execute() {
         ArrayList<String> foundedFinancialList = new ArrayList<>();
         ArrayList<String> foundedWatchList = new ArrayList<>();
         for (int i = 0; i < cashflowList.list.size(); i++) {
-            if (cashflowList.list.get(i).formatString().contains(description)) {
-                foundedFinancialList.add(cashflowList.list.get(i).formatString());
+            if (cashflowList.list.get(i).toString().contains(description)) {
+                String output = cashflowList.list.get(i).toString()+" | Index: "+(i+1);
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/commands/FindCommand.java:29]: good job on keeping the code length short

**Review by NeoMinWei — COMMENTED** [review]
LGTM

**Pull request merged**